src/node_crypto.cc

```diff
@@ -3952,6 +3954,37 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment* env, v8::Local<v8::Object> options,
```

Comment:
Positional arguments are easier and more efficient than wrangling objects in C++ land.

As well, it would get rid of the ThrowError().  Validation code that throws JS exceptions as a side effect tends to be pretty brittle.  Callers need to be very careful not to re-enter the VM on return but that is easy to get wrong.